src/query/storage/m3/dynamic_cluster_test.go

```diff
@@ -423,8 +460,8 @@ func assertClusterNamespace(clusters Clusters, expectedID ident.ID, expectedOpts
 		}); !ok {
 			return false
 		}
-	} else {
-		ns = clusters.UnaggregatedClusterNamespace()
+	} else if ns, ok = clusters.UnaggregatedClusterNamespace(); !ok {
```

Comment:
Can we have this as a separate if? Otherwise `ns` almost seems like it's getting set as a side effect
```suggestion
	} 
	
	ns, ok = clusters.UnaggregatedClusterNamespace()
	if !ok {
```

Reply:
needs to be in an else, will do